RecoEcal/EgammaClusterProducers/src/PFECALSuperClusterProducer.cc

```diff
@@ -50,9 +50,12 @@ PFECALSuperClusterProducer::PFECALSuperClusterProducer(const edm::ParameterSet&
     iConfig.getUntrackedParameter<bool>("verbose",false);
 
   superClusterAlgo_.setUseRegression(iConfig.getParameter<bool>("useRegression")); 
-    
+
+  isOOTCollection_ = (iConfig.existsAs<bool>("isOOTCollection") ? iConfig.getParameter<bool>("isOOTCollection") : false);
```

Comment:
"iConfig.existsAs" is not needed if the value is already inserted in the fillDescriptions.
Please remove.